integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/mapping/ElasticsearchTypeNameMappingSchemaIT.java

```diff
@@ -126,8 +128,17 @@ private JsonObject indexCreationPayload() {
 
 		payload.add( "mappings", mappings );
 
-		payload.add( "settings", new JsonObject() );
+		payload.add( "settings", getSettings() );
 
 		return payload;
 	}
+
+	private JsonObject getSettings() {
+		JsonObject settings = new JsonObject();
+		settings.add(
+				IndexSettings.MAX_RESULT_WINDOW_ATTRIBUTE,
+				new JsonPrimitive( ElasticsearchIndexSettings.Defaults.MAX_RESULT_WINDOW_SIZE )
+		);
+		return settings;
+	}
```

Comment:
... Why is this necessary? You shouldn't have to set the defaults, they are implicit.